packages/block-library/src/archives/index.php

```diff
@@ -90,9 +88,7 @@ function render_block_core_archives( $attributes ) {
 
 	$archives = wp_get_archives( $archives_args );
 
-	$classnames = esc_attr( $class );
-
-	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $classnames ) );
+	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => esc_attr( $class ) ) );
```

Comment:
It looks like this will result in double escaping since `get_block_wrapper_attributes` already escapes attribute values when rendering to a string.

Reply:
I think #37985 addressed similar issues.

Reply:
👍 Thanks @Mamaduka! Sorry for missing that 🤦.

Reply:
Yeh we later decided it was ok to rely on these functions.